include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3604,6 +3604,11 @@ struct OrtApi {
    *   "qnn_saver_path": File path to the QNN Saver backend library. If specified, QNN Saver will be enabled and will
    *   dump QNN API calls to disk for replay/debugging. QNN Saver produces incorrect model inference results and
    *   may alter model/EP partitioning. Use only for debugging.
+   *   "htp_graph_finalization_optimization_mode": Set the optimization mode for graph finalization on the HTP backend. Available options:
```

Comment:
>htp_graph_finalization_optimization_mode [](start = 8, length = 40)

Need to enable this with the onnxruntime_perf_test tool.

Reply:
Added, thanks